main/src/main/java/com/ldtteam/blockout/render/standard/RenderingController.java

```diff
@@ -0,0 +1,641 @@
+package com.ldtteam.blockout.render.standard;
+
+import com.ldtteam.blockout.element.IUIElement;
+import com.ldtteam.blockout.element.simple.Slot;
+import com.ldtteam.blockout.gui.BlockOutGuiData;
+import com.ldtteam.blockout.management.render.IRenderManager;
+import com.ldtteam.blockout.render.core.IRenderingController;
+import com.ldtteam.blockout.render.core.IScissoringController;
+import com.ldtteam.blockout.util.color.IColor;
+import com.ldtteam.blockout.util.math.BoundingBox;
+import com.ldtteam.blockout.util.math.Vector2d;
+import com.ldtteam.jvoxelizer.IGameEngine;
+import com.ldtteam.jvoxelizer.biome.IBiome;
+import com.ldtteam.jvoxelizer.block.state.IBlockState;
+import com.ldtteam.jvoxelizer.client.gui.IGuiContainer;
+import com.ldtteam.jvoxelizer.client.renderer.bufferbuilder.IBufferBuilder;
+import com.ldtteam.jvoxelizer.client.renderer.font.IFontRenderer;
+import com.ldtteam.jvoxelizer.client.renderer.item.IItemRenderer;
+import com.ldtteam.jvoxelizer.client.renderer.opengl.IOpenGl;
+import com.ldtteam.jvoxelizer.client.renderer.opengl.util.DestinationFactor;
+import com.ldtteam.jvoxelizer.client.renderer.opengl.util.SourceFactor;
+import com.ldtteam.jvoxelizer.client.renderer.opengl.util.vertexformat.IVertexFormat;
+import com.ldtteam.jvoxelizer.client.renderer.tessellator.ITessellator;
+import com.ldtteam.jvoxelizer.client.renderer.texture.ISprite;
+import com.ldtteam.jvoxelizer.client.renderer.texture.ISpriteMap;
+import com.ldtteam.jvoxelizer.core.logic.DummyInstanceData;
+import com.ldtteam.jvoxelizer.dimension.IDimensionReader;
+import com.ldtteam.jvoxelizer.dimension.IDimensionType;
+import com.ldtteam.jvoxelizer.dimension.logic.builder.IDimensionReaderBuilder;
+import com.ldtteam.jvoxelizer.fluid.IFluidStack;
+import com.ldtteam.jvoxelizer.inventory.IContainer;
+import com.ldtteam.jvoxelizer.inventory.slot.ISlot;
+import com.ldtteam.jvoxelizer.item.IItemStack;
+import com.ldtteam.jvoxelizer.util.identifier.IIdentifier;
+import com.ldtteam.jvoxelizer.util.math.coordinate.block.IBlockCoordinate;
+import com.ldtteam.jvoxelizer.util.textformatting.ITextFormatting;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class RenderingController implements IRenderingController
```

Comment:
This class appears to contain a lot of not yet deobfuscated names and inlined opengl constants

Reply:
Yep will try to find the constants.